tunix/sft/metrics_logger.py

```diff
@@ -24,6 +24,7 @@
 
 @dataclasses.dataclass
 class MetricsLoggerOptions:
+  """Metrics Logger options."""
```

Comment:
A stub docstring to prevent the listing all input parameters in the landing page.